Containers/talk/start.sh

```diff
@@ -30,14 +30,21 @@ if [ -n "$IPv4_ADDRESS_TALK" ] && [ "$IPv4_ADDRESS_TALK_RELAY" = "$IPv4_ADDRESS_
     IPv4_ADDRESS_TALK=""
 fi
 
+IP_BINDING="::"
+if grep -q "1" /sys/module/ipv6/parameters/disable \
+|| sysctl -a 2>/dev/null | grep "net.ipv6.conf.all.disable_ipv6" | grep -q "1" \
+|| sysctl -a 2>/dev/null | grep "net.ipv6.conf.default.disable_ipv6" | grep -q "1"; then
```

Comment:
Not sure if we actually need all of these checks or if the check for /sys/module/ipv6/parameters/disable is enough?

Reply:
I'm also not sure but I would keep them all. The first one would be to check if ipv6 was disabled via boot args and the second/third ones via sysctl.

Reply:
I'm also not sure but I would keep them all. The first one would be to check if ipv6 was disabled via boot args and the second/third ones via sysctl.

Reply:
okay